libsawtooth/Cargo.toml

```diff
@@ -24,6 +24,7 @@ description = """\
 """
 
 [dependencies]
+base64 = "0.12.3"
```

Comment:
This should be optional and added to the `client-rest` feature's dependent features.

Reply:
Fixed